dirsrvtests/tests/suites/logging/logconv_test.py

```diff
@@ -0,0 +1,980 @@
+        """
+        
+        if not self.logconv_path:
+            raise FileNotFoundError(f"{self.logconv_path} not found.")
+
+        if not self.access_log_path:
+            raise FileNotFoundError(f"{self.access_log_path} not found.")
+        
+        try:
+            result = subprocess.run(
+                [self.logconv_path, self.access_log_path],
+                capture_output=True,
+                text=True
+            )
+        except FileNotFoundError as e:
+            print("Exception message 1:", e.value)
+            raise FileNotFoundError(f"Logconv script not found: {self.logconv_path}") from e
+        except Exception as e:
+             print("Exception message 2:", e.value)
+             raise RuntimeError(f"Logconv script unexpected error: {e}") from e
+
+        if result.returncode != 0:
+            raise RuntimeError(f"{self.logconv_path} failed: {result.stderr}")
+
+        summary = result.stdout
+
+        if debug_output:
+            with open(self.debug_output, 'w') as f:
+                f.write(summary)
+            os.chmod(self.debug_output, 0o644)
+
+        return summary
+
+    def wait_for_log_counts(self, expected: dict, test_name=None, poll_interval=1):
+        """
+        Wait for the parsed logconv output to meet or exceed the expected op count, 
+        retrying every poll_interval until the condition is met or it times out.
+
+        The comparison is loose, it passes once the actual counts are greater than or equal 
+        to the expected count.
+
+        Args:
+            expected: dict of expected exact counts
+            test_name: string for logging
+            poll_interval: how often to poll the logs in seconds
+        """
+        IGNORE_KEYS = {"highest_fd_taken", "concurrent_conns"}
+
+        now = time.time()
+        while True:
+            errors = []
+            try:
+                output = self.run_logconv()
+            except (FileNotFoundError, RuntimeError) as e:
+                log.error(f"{test_name}: Error running logconv: {e}")
+                raise
+
+            summary = self.parse_summary(output)
+            for key, existing_val in summary.items():
+                if key in IGNORE_KEYS:
+                    continue
+                expected_val = expected.get(key, 0)
+                if existing_val < expected_val:
+                    errors.append(f"{test_name} - {key}: expected {expected_val}, got {existing_val}")
+
+            if not errors:
+                log.info(f"{test_name}: Pass.")
+                return
+
+            if (time.time() - now) > self.TEST_TIMEOUT:
+                error_report = "\n".join(errors)
+                return f"\n{error_report}"
+ 
+            time.sleep(poll_interval)
+
+@pytest.fixture
+def fresh_instance():
+    topo = create_topology({ReplicaRole.STANDALONE: 1})
+    inst = topo.standalone
+    inst.start()
+    yield inst
+    inst.stop()
+    inst.delete(escapehatch="i am sure")
```

Comment:
It needs a finalizer.

So, I think, it's better to reuse a usual topology approach:

```python


@pytest.fixture(scope="function")
def topology_st(request):
    """Create DS standalone instance"""

    topology = create_topology({ReplicaRole.STANDALONE: 1})

    def fin():
        if topology.standalone.exists():
            topology.standalone.delete()
    request.addfinalizer(fin)

    return topology